packages/components/organisms/f-checkout/src/services/addressGeocodingApi.js

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+
+export default {
+    async getGeoLocation (url, postData, timeout, state) {
+        const authHeader = state.authToken && `Bearer ${state.authToken}`;
```

Comment:
THis could be easier to read as:
```javascript
if (state.authToken)
    authHeader = `Bearer ${state.authToken}`
else 
    authToken = '';
```

Reply:
I personally don't agree with this comment. It's a matter of personal preference, in the end, but we've been using this line across the project.

Reply:
will leave as is for now :)